### DISCUSSION TITLE
Add detection for used import in the html portion of svelte

### DISCUSSION THREAD

**Discussion by LazyYuuki** [biomejs/biome#2212] [Ideas]
I want to ask if there is a plan to add the feature to help check for usage of import that is not used in the <script> tag portion but used in the html portion of Svelte instead. For example:

```js
<script>
  import { foo } from "./bar"  // Unused import linter warning
</script>

<div>
  {foo} // import is used here
</div>
```

If there is already said feature, how do I go about setting it up? I am currently on neovim using biome as the default LSP, Linter and Formatter. Any help is much appreciated.

**Comment by notramo** [comment]
The markup part is not supported yet. You can disable that rule in the Biome config. I recommend adding a TODO to the config to explain that it's disabled because unsupported, and to re-enable it once it's implemented.
